src/core/education/service/education.service.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  BadRequestException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
+import { CreateEducationDto } from '../dto/create-education.dto';
+import { UpdateEducationDto } from '../dto/update-education.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Education } from '../entities/education.entity';
+import { Repository } from 'typeorm';
+import { UserActiveInterface } from 'src/common/interface/user-active-interface';
+import {
+  EDUCATION_NOT_FOUND,
+  ERROR_EDUCATION_IS_UCAB,
+  ERROR_EDUCATION_PRINCIPAL_ALREADY_EXISTS,
+} from '../message';
+
+@Injectable()
+export class EducationService {
+  constructor(
+    @InjectRepository(Education)
+    private educationRepository: Repository<Education>,
+  ) {}
+
+  async create(
+    createEducationDto: CreateEducationDto,
+    user: UserActiveInterface,
+  ) {
+    const newEducation = await this.educationRepository.save({
+      ...createEducationDto,
+      profileId: user.id,
+    });
```

Comment:
Debe estar quemado que toda experiencia que puedas crear NO es de la UCAB
```suggestion
      profileId: user.id,
      isUCAB: false
    });
```